lib/power_action_utils.pm

```diff
@@ -206,15 +206,6 @@ sub poweroff_x11 {
         assert_screen 'logout-confirm-dialog', 10;
         send_key "alt-o";              # _o_k
     }
-
-    if (check_var('BACKEND', 's390x')) {
-        # make sure SUT shut down correctly
-        console('x3270')->expect_3270(
-            output_delim => qr/.*SIGP stop.*/,
-            timeout      => 30
-        );
-
-    }
```

Comment:
@OleksandrOrlov i cant really understand why you need to remove this, as you can just prevent the code to reach this point using the textmode parameter when the backend is s390x, so it will just behave as before.

Reply:
@b10n1k I've removed the code as it shouldn't be located here. It was a hack, which also was not executed for a long time.

`poweroff_x11` implements power off actions for all the architectures, but does not do assert shutdown for any of them. For s390x it was added here, probably as quick workaround. 

I removed the code and created a follow-up ticket to implement working solution for asserting shutdown on s390x backend and using same design as for all other backends (implement `is_shutdown` function in os-autoinst).